src/SeqCli/PlainText/Extraction/Matchers.cs

```diff
@@ -17,42 +17,47 @@ static class Matchers
         [Matcher("ident")]
         public static TextParser<object> Identifier { get; } =
             Superpower.Parsers.Identifier.CStyle
-                .Select(span => (object) span);
+                .Cast<TextSpan, object>();
         
         [Matcher("nat")]
         public static TextParser<object> Natural { get; } =
             Numerics.NaturalUInt64
-                .Select(span => (object) span);
+                .Cast<ulong, object>();
         
         [Matcher("int")]
         public static TextParser<object> Integer { get; } =
             Numerics.IntegerInt64
-                .Select(span => (object) span);
+                .Cast<long, object>();
         
         [Matcher("dec")]
         public static TextParser<object> Decimal { get; } =
             Numerics.Decimal
-                .Select(span => (object) span);
+                .Cast<TextSpan, object>();
         
         [Matcher("alpha")]
         public static TextParser<object> Alphabetical { get; } =
             Span.WithAll(char.IsLetter)
-                .Select(span => (object) span);
+                .Cast<TextSpan, object>();
         
         [Matcher("alphanum")]
         public static TextParser<object> Alphanumeric { get; } =
             Span.WithAll(char.IsLetterOrDigit)
-                .Select(span => (object) span);
+                .Cast<TextSpan, object>();
         
         [Matcher("token")]
         public static TextParser<object> Token { get; } =
-            SpanEx.NonWhiteSpace.Select(span => (object)span);
+            SpanEx.NonWhiteSpace.Cast<TextSpan, object>();
+
+        [Matcher("s")]
+        public static TextParser<object> WhiteSpace { get; } =
+            Span.WithAll(ch => ch == ' ' || ch == '\t')
```

Comment:
Would you expect this to match carriage returns and linefeeds too?

Reply:
Many of the other patterns are linefeed-aware, so slurping them up can have unexpected consequences.